polybar-scripts/inbox-reddit/inbox-reddit.sh

```diff
@@ -1,10 +1,31 @@
 #!/bin/sh
+#
+# This script retrieves unread messages from your Reddit account using a JSON
+# link.
+#
+# To use this script, make sure you have the following dependencies installed:
+# `curl`, `jq`, `ttf-font-awesome`.
+#
+# NOTE: `inbox-reddit` can also be used outside of polybar by replacing the
+# following line: echo -e "%{F#cb4b16} %{F-}$unread"
+# by: echo " $unread"
+#
+# WARNING: in order to easily share this script without security risk, it is
+# recommended to place the JSON part containing the sensitive information in a
+# separate GPG file.
 
-url="your url here"
+# Replace this path by yours.
+reddit_gpg="/home/$USER/Sync/shared/.reddit-json.gpg"
+url="https://www.reddit.com/message/unread/.json?feed=$(gpg -qd $reddit_gpg)"
 unread=$(curl -sf "$url" | jq '.["data"]["children"] | length')
 
+case "$unread" in
+    ''|*[!0-9]*)
+	unread=0
+esac;
+
 if [ "$unread" -gt 0 ]; then
-   echo "#1 $unread"
+    echo -e "%{F#cb4b16} $unread%{F-}"
```

Comment:
Please take a look at the [CONTRIBUTING.md](https://github.com/x70b1/polybar-scripts/blob/master/CONTRIBUTING.md).

There is a reason why we don't want Icons and colors in our scripts. The scripts should be easy to handle. If somebody wan't colors or icons he can change the script parts.